### PR TITLE
Real logging support via spec file

### PR DESCRIPTION
**Pull request by mwaldrich**

### CODE REVIEW

File: tests-unit/README.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ The test specification file must be of the form:
     // The script to execute
     main: string;
     
+    // Enable verbose logging
+    verbose?: boolean;
```
Comment (trueblooda) [inline tests-unit/README.md:42]: Boolean? More Like Verbose!!!!

File: ts/src/abstractMachine/logger.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import * as bunyan from "bunyan";
-export default bunyan.createLogger({
```
Comment (trueblooda) [inline ts/src/abstractMachine/logger.ts:2]: why are you removing Paul Bunyan from this????

File: ts/src/abstractMachine/logger.ts [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
-    level: "fatal",
```
Comment (trueblooda) [inline ts/src/abstractMachine/logger.ts:3]: Good thing your not on fatal status anymore!!!

File: ts/src/utils.ts [hunk 4]
```diff
@@ -175,10 +175,13 @@ export function parseSpec(specPath: string): RunSpecification {
         process.exit(1);
     }
 
-    // Add empty sanitizers array if the user didn't specify any
```
Comment (trueblooda) [inline ts/src/utils.ts:178]: Please refill the sanitizers. Im not interested in Sars-Covid-19.

File: tests-unit/README.md (hunk 1 above)
Comment (mwaldrich) [reply tests-unit/README.md:42]: yes

File: ts/src/abstractMachine/logger.ts (hunk 3 above)
Comment (mwaldrich) [reply ts/src/abstractMachine/logger.ts:3]: ikr!

File: ts/src/abstractMachine/logger.ts (hunk 2 above)
Comment (trueblooda) [reply ts/src/abstractMachine/logger.ts:2]: > there's no place for lumberjacks in this codebase sir
^ this is what you sound like

File: ts/src/abstractMachine/logger.ts (hunk 3 above)
Comment (mwaldrich) [reply ts/src/abstractMachine/logger.ts:3]: 😌

File: ts/src/utils.ts (hunk 4 above)
Comment (mwaldrich) [reply ts/src/utils.ts:178]: the virus is actually sars-cov-2, which causes the covid-19 respiratory infection

**Review by trueblooda — APPROVED** [review]

**Pull request merged**